modules/bundle/README.md

```diff
@@ -0,0 +1,9 @@
+# MONAI bundle
+This folder contains the get_started tutorial and concrete training / inference examples for MONAI bundle features.
+
+### [spleen segmentation](./spleen_segmentation)
+A bundle example for volumetric (3D) segmentation of the spleen from CT image.
+### [customize component](./custom_component)
+Example shows the use case that brings customized python component(like: transform, network, metrics, etc.) in the `train` config.
```

Comment:
```suggestion
Example shows the use cases of bringing customized python components, such as transform, network, and metrics, in a configuration-based workflow.
```

Reply:
updated in the latest commit.

Thanks.